### ISSUE TITLE
Prowlarr RED Indexer Not Working

### ISSUE THREAD

**Issue by tehfonz** [Prowlarr/Prowlarr#2449]
### Is there an existing issue for this?

- [x] I have searched the existing open and closed issues

### Current Behavior

RED Indexer times out
1. RED account is active and enabled
2. API key is valid
3. All other trackers are connecting just fine
4. Confirmed with Admin that there is no block or ban on my IP

### Expected Behavior

RED Indexer should connect

### Steps To Reproduce

Add Indexer -> RED -> Change base URL to only available option -> paste API Key

### Environment

```markdown
- OS: Debian 12
- Prowlarr: 1.37.0.5076
- Docker Install: No
- Using Reverse Proxy: No
- Browser: Any
```

### What branch are you running?

Master

### Trace Logs?

2025-07-23 08:49:59.0|Trace|Http|Req: 29335 [POST] /api/v1/indexer/test (from 192.168.1.10 Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:140.0) Gecko/20100101 Firefox/140.0)
2025-07-23 08:49:59.0|Debug|Prowlarr.Http.Authentication.ApiKeyAuthenticationHandler|AuthenticationScheme: API was successfully authenticated.
2025-07-23 08:49:59.0|Debug|Redacted|Downloading Feed https://redacted.sh/ajax.php?action=browse&order_by=time&order_way=desc
2025-07-23 08:49:59.0|Trace|ConfigService|Using default config value for 'logindexerresponse' defaultValue:'False'
2025-07-23 08:49:59.0|Trace|IndexerHttpClient|Req: [GET] https://redacted.sh/ajax.php?action=browse&order_by=time&order_way=desc
2025-07-23 08:49:59.0|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2025-07-23 08:50:01.9|Trace|Scheduler|Pending Tasks: 0
2025-07-23 08:50:31.9|Trace|Scheduler|Pending Tasks: 0
2025-07-23 08:51:01.9|Trace|Scheduler|Pending Tasks: 0
2025-07-23 08:51:31.9|Trace|Scheduler|Pending Tasks: 0
2025-07-23 08:51:39.0|Warn|Redacted|Unable to connect to indexer.

[v1.37.0.5076] System.Net.WebException: Http request timed out
 ---> System.Threading.Tasks.TaskCanceledException: A task was canceled.
   at System.Net.Http.HttpClient.HandleFailure(Exception e, Boolean telemetryStarted, HttpResponseMessage response, CancellationTokenSource cts, CancellationToken cancellationToken, CancellationTokenSource pendingRequestsCts)
   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)
   at NzbDrone.Common.Http.Dispatchers.ManagedHttpDispatcher.GetResponseAsync(HttpRequest request, CookieContainer cookies) in ./Prowlarr.Common/Http/Dispatchers/ManagedHttpDispatcher.cs:line 120
   --- End of inner exception stack trace ---
   at NzbDrone.Common.Http.Dispatchers.ManagedHttpDispatcher.GetResponseAsync(HttpRequest request, CookieContainer cookies) in ./Prowlarr.Common/Http/Dispatchers/ManagedHttpDispatcher.cs:line 170
   at NzbDrone.Common.Http.HttpClient.ExecuteRequestAsync(HttpRequest request, CookieContainer cookieContainer) in ./Prowlarr.Common/Http/HttpClient.cs:line 171
   at NzbDrone.Common.Http.HttpClient.ExecuteAsync(HttpRequest request) in ./Prowlarr.Common/Http/HttpClient.cs:line 70
   at NzbDrone.Core.Indexers.IndexerHttpClient.ExecuteProxiedAsync(HttpRequest request, ProviderDefinition definition) in ./Prowlarr.Core/Indexers/IndexerHttpClient.cs:line 44
   at NzbDrone.Core.Indexers.HttpIndexerBase`1.<>c.<<FetchIndexerResponse>b__57_0>d.MoveNext() in ./Prowlarr.Core/Indexers/HttpIndexerBase.cs:line 665
--- End of stack trace from previous location ---
   at Polly.ResiliencePipeline.<>c__9`2.<<ExecuteAsync>b__9_0>d.MoveNext()
--- End of stack trace from previous location ---
   at Polly.Outcome`1.GetResultOrRethrow()
   at Polly.ResiliencePipeline.ExecuteAsync[TResult,TState](Func`3 callback, TState state, CancellationToken cancellationToken)
   at NzbDrone.Core.Indexers.HttpIndexerBase`1.FetchIndexerResponse(IndexerRequest request) in ./Prowlarr.Core/Indexers/HttpIndexerBase.cs:line 664
   at NzbDrone.Core.Indexers.HttpIndexerBase`1.FetchPage(IndexerRequest request, IParseIndexerResponse parser) in ./Prowlarr.Core/Indexers/HttpIndexerBase.cs:line 610
   at NzbDrone.Core.Indexers.HttpIndexerBase`1.TestConnection() in ./Prowlarr.Core/Indexers/HttpIndexerBase.cs:line 856

2025-07-23 08:51:39.1|Trace|ProwlarrErrorPipeline|Handling Exception
2025-07-23 08:51:39.1|Warn|ProwlarrErrorPipeline|Invalid request Validation failed: 
 -- : Unable to connect to indexer, indexer's server is unavailable. Try again later. Http request timed out
2025-07-23 08:51:39.1|Trace|Http|Res: 29335 [POST] /api/v1/indexer/test: 400.BadRequest (100107 ms)
2025-07-23 08:51:39.1|Debug|Api|[POST] /api/v1/indexer/test: 400.BadRequest (100107 ms)

### Trace Logs have been provided as applicable. Reports may be closed if the required logs are not provided.

- [x] I have read and followed the steps in the wiki link above and provided the required trace logs - the logs contain `trace` - that are relevant and show this issue.

**Comment by bakerboy448** [comment]
Timeouts are not bugs

**Comment by tehfonz** [comment]
Thanks bakerboy! I'll jump in discord and see what's what. Cheers!
